<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Samples</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="imageWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_File">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionOpen_folder"/>
    <addaction name="actionSave_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionFolder_detection"/>
    <addaction name="actionCulc_Stat"/>
    <addaction name="actionChange_view"/>
   </widget>
   <addaction name="menuOpen_File"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_File"/>
   <addaction name="actionSave_File"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionFolder_detection"/>
   <addaction name="actionCulc_Stat"/>
   <addaction name="actionChange_view"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/fileopen.png</normaloff>:/new/prefix1/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open Files</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/filesave-48.png</normaloff>:/new/prefix1/Icons/filesave-48.png</iconset>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/Gnome-Application-Exit.png</normaloff>:/new/prefix1/Icons/Gnome-Application-Exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/Help.png</normaloff>:/new/prefix1/Icons/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/run-icon.png</normaloff>:/new/prefix1/Icons/run-icon.png</iconset>
   </property>
   <property name="text">
    <string>Detection</string>
   </property>
   <property name="toolTip">
    <string>Run Classifier</string>
   </property>
  </action>
  <action name="actionPaint_Objects">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/Ballon-white.ico</normaloff>:/new/prefix1/Icons/Ballon-white.ico</iconset>
   </property>
   <property name="text">
    <string>Paint Objects</string>
   </property>
  </action>
  <action name="actionRubber">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/Eraser-icon.png</normaloff>:/new/prefix1/Icons/Eraser-icon.png</iconset>
   </property>
   <property name="text">
    <string>Rubber</string>
   </property>
  </action>
  <action name="actionRun_thres">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/blue-rectangle-icone-8818-128.png</normaloff>:/new/prefix1/Icons/blue-rectangle-icone-8818-128.png</iconset>
   </property>
   <property name="text">
    <string>Run Threshold Classification</string>
   </property>
  </action>
  <action name="actionCuculate_thres">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/128x128-18-2120.png</normaloff>:/new/prefix1/Icons/128x128-18-2120.png</iconset>
   </property>
   <property name="text">
    <string>Cuculate Threshold</string>
   </property>
  </action>
  <action name="actionCulc_Stat">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/Box habanos open.png</normaloff>:/new/prefix1/Icons/Box habanos open.png</iconset>
   </property>
   <property name="text">
    <string>Select Area</string>
   </property>
  </action>
  <action name="actionChange_view">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/blue-rectangle-icone-8818-128.png</normaloff>:/new/prefix1/Icons/blue-rectangle-icone-8818-128.png</iconset>
   </property>
   <property name="text">
    <string>Change_view</string>
   </property>
  </action>
  <action name="actionOpen_folder">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/box_open.png</normaloff>:/new/prefix1/Icons/box_open.png</iconset>
   </property>
   <property name="text">
    <string>Open folder</string>
   </property>
   <property name="toolTip">
    <string>Open all images inside folder</string>
   </property>
  </action>
  <action name="actionFolder_detection">
   <property name="icon">
    <iconset resource="sources.qrc">
     <normaloff>:/new/prefix1/Icons/box_open.png</normaloff>:/new/prefix1/Icons/box_open.png</iconset>
   </property>
   <property name="text">
    <string>Folder detection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
